### ISSUE TITLE
Help/advice on the autobuild

### ISSUE THREAD

**Issue by theincognito-inc** [AnimMouse/ffmpeg-autobuild#85]
Hi,

I am coming here from reddit. First of all, thanks for providing binaries where zeranoe left off! 😄 

Now, I am not a noob and I am fairly good with git. But GitHub actions are beyond my expertise. Your binaries provide everything I want, except one tiny patch. I can compile ffmpeg with that patch locally, but my PC is potato (😭) and so can't update to the newer git versions with that patch everytime.
From what I can understand (feel free to correct me if I am wrong), I could clone/fork your repo and use it for autobuild. But I dunno how to apply my patch before building.

TL;DR - Help on using autobuild to build ffmpeg binaries like you do. After fetching the latest ffmpeg source, need to apply a custom patch before building each time.

**Comment by AnimMouse** [comment]
> I am coming here from reddit. First of all, thanks for providing binaries where zeranoe left off! 😄

You're welcome.

How did you compile FFmpeg with your own patch? Can you provide instructions on how to compile your own version of FFmpeg with your patch? So I can try to translate it in GitHub Actions. Custom compilation is also beyond my expertise, that is why I use ffmpeg-windows-build-helpers instead of making my own toolchain.

> From what I can understand (feel free to correct me if I am wrong), I could clone/fork your repo and use it for autobuild. But I dunno how to apply my patch before building.

You are right, you can do it, but I use ffmpeg-windows-build-helpers, did you use ffmpeg-windows-build-helpers in order to compile your FFmpeg with your patch? Or your own toolchain?

Is that "patch" you are calling are a git patch? If yes, it seems like we can apply it to the submodule before compilation.

**Comment by theincognito-inc** [comment]
It's just a git patch. An one line change, really. An acquaintance of mine created the patch, he even submitted it to FFMPEG team, they ignored it. Basically changing the header size from 4096 to 8192. That's all. 
[git-patch.zip](https://github.com/AnimMouse/ffmpeg-autobuild/files/5314225/git-patch.zip)
I have uploaded the patch, as-is, above. If you can integrate it, that would be cool too. It doesn't break anything, it just increases the header size.

**Comment by AnimMouse** [comment]
1. Fork this repo
2. Enable GitHub Actions on your fork. GitHub disables GitHub Actions in forks by default.
3. Download this patch and apply it to your own fork. Your patch is included in my patch.
[0001-Add-application-of-patches.patch.gz](https://github.com/AnimMouse/ffmpeg-autobuild/files/5322064/0001-Add-application-of-patches.patch.gz)
4. Push, and let it compile.
I have added a code that will apply all the patches in the `ffmpeg-patches` directory.

> An acquaintance of mine created the patch, he even submitted it to FFMPEG team, they ignored it. Basically changing the http header size from 4096 to 8192. That's all.

Did the FFmpeg team reply? Or they just silently ignored it? Tell your acquaintance to resubmit it again. Probably they were overwhelmed by the number of patches they receive every day.

**Comment by theincognito-inc** [comment]
Done. Thanks a lot, mate!!! Will ask him to resubmit the patch 😄 
Thanks again, for everything 😃

**Comment by AnimMouse** [comment]
You're welcome.
